pkg/cmd/triggertemplate/list_test.go

```diff
@@ -0,0 +1,139 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package triggertemplate
```

Comment:
In the future, I think this, along with all our tests that follow this format, should be refactored to follow the pattern such as with [taskrun ls](https://github.com/tektoncd/cli/blob/369ab71f9ffbdf279a34aabaa18ee2c93dfebbe8/pkg/cmd/taskrun/list_test.go#L111). But no worries for now.

Reply:
I think that won't make much difference and we can fix later if required for some use-case..